impls/msft-green-ai.yaml

```diff
@@ -0,0 +1,44 @@
+name: mft-green-ai
+description: "https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/msft-green-ai.md"
+tags:
+  kind: ml
+  complexity: simple
+  category: cloud
+config:
+  funit:
+    - name: job
+      scale-factor: 1 # divide SCI by this value to get SCI per ml job
+  models:
```

Comment:
The case study talks about using watttime data. I think we can imagine here also including a model called `watttime` that calls the wattime service for a particual time and region to get the grid-ci value to add to the observation.